test/integration/admin/new_announcements_test.rb

```diff
@@ -1,30 +1,17 @@
 require 'test_helper'
 
 class Admin::NewAnnouncementsTest < ActionDispatch::IntegrationTest
-  test 'an admin user can create a new announcement' do
-    user = create(:user, :admin)
-
-    get new_admin_announcement_path
-    assert_template 'admin/announcements/new'
-    title = 'Title'
-    body = 'Body'
-    expiration_date = Date.new(2015,5,5)
-    created_by = 'Admin'
-    assert_difference 'Announcement.count', 1 do
-      post_via_redirect admin_announcements_path, { announcement:
-                                      { title: title,
-                                        body: body,
-                                        expiration_date: expiration_date,
-                                        created_by: created_by
-                                      }
-                                    }
-    end
-    refute flash.empty?
-    assert_template 'admin/announcements/index'
+  def login_admin
+    FactoryGirl.create(:user, :admin)
+    visit '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.